README.md

```diff
@@ -1,97 +1,27 @@
-# HTML & CSS3 Course
+#HELLO MICROVERSE
 
-If you are not familiar with linters, read [root level README](../README.md).
+    Description the project.
+    In this project, you will set up a "Hello Microverse" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
 
-## Set-up GitHub Actions
 
-Please do the following **steps in this order**:
+#Built With
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+    Major languages : HTML&CSS, linter
+    Frameworks : none
+    Technologies used : Git, Github
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+#Requirements
 
-Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+   Remember to document your work in a professional way.
+   Create a new GitHub repo.
+   Create a new branch according to the GitHub flow rules.
+   Add a .gitignore file.
+   Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+   Add descriptive README file to your project - please use this template but remember to customize it to your project.
+   Add 3 files (each one in a separate commit):
+   A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
+   A .css file that includes one class with styles for your header (make it in your favorite color).
+   A test.md file with the text "This file should be ignored by git".
+   Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
+   Open a pull request.
 
```

Comment:
- [ ] Kindly use Microverse template and update your README file- please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. Just Take it cope from [here](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) and paste it to your readme file.